app/pods/courses/index/controller.js

```diff
@@ -1,31 +1,21 @@
 import Ember from 'ember';
-import { computed } from '@ember/object';
 import { task } from 'ember-concurrency';
 
 export default Ember.Controller.extend({
   queryParams: ['limit', 'offset'],
   limit: 9,
   offset: 0,
 
-
-  showLoadMore: computed('offset', 'count', 'limit', function () {
-    let limit = this.get('limit')
-    let offset = this.get('offset')
-    let count = this.get('count')
-    return offset + limit < count
-  }),
-
   taskMoreCourses: task(function * () {
     const nextCourses = yield this.store.query ('course', {
       include: 'runs',
       sort: 'difficulty',
       filter: {
         unlisted: false
       },
-      page:{
-        limit: this.get('limit'),
-        offset: this.get('offset')
-      }
+      limit: this.get('limit'),
```

Comment:
shouldn't this be inside page?
backend api expects `page[offset]` and `page[limit]`